tripal/src/Services/TripalPublish.php

```diff
@@ -477,9 +477,10 @@ protected function getEntityTitles($matches) {
           $field = $this->field_info[$field_name]['instance'];
           $main_prop = $field->mainPropertyName();
           $value = $match[$field_name][$delta][$main_prop]['value']->getValue();
```

Comment:
```suggestion
          $value = '';
          if (array_key_exists($delta, $match[$field_name])) {
            $value = $match[$field_name][$delta][$main_prop]['value']->getValue();
          }
```
With this change I can manually publish an organism, and run the failing test locally without error.